core/field_dropdown.ts

```diff
@@ -44,7 +44,9 @@ export class FieldDropdown extends Field {
    * height.
    */
   static MAX_MENU_HEIGHT_VH = 0.45;
-  static ARROW_CHAR: AnyDuringMigration;
+
+  /** Android can't (in 2014) display "▾", so use "▼" instead. */
+  static ARROW_CHAR: string = userAgent.ANDROID ? '▼' : '▾';
```

Comment:
Pretty sure this isn't actually a problem anymore (I can view this file just fine on my android phone). Could you have this just use the small carrot string + remove the comment?

Reply:
Done!